src/redux/actions/samples/saveSamples.js

```diff
@@ -31,10 +31,7 @@ const saveSamples = (
     });
 
     payload = projectSamples;
-    payload = {
-      ...payload,
-      [newSample.uuid]: newSampleToUpload,
-    };
+    payload[newSample.uuid] = newSampleToUpload;
```

Comment:
why removing the `...payload`?

Reply:
These lines insert the new sample uuid and sample object to the payload. In the previous implementation we create a new object by spreading the payload object and inserting the key. In this new implementation, we just insert the sample record to the payload object with a new key, without having to create a new object. So this new implementation is more efficient.